client/app/utils/transitImmutableConverter.js

```diff
@@ -16,6 +16,10 @@ const toImage = (data) => {
   return styles;
 };
 
+// Outside of this file we should only pass UUID references, no need to export
+const UUID = Immutable.Record({ stringRep: '' });
+const toUUID = (transitUuid) => new UUID({ stringRep: transitUuid.toString() });
```

Comment:
If Record wrapper is what we want, then also ListingModel's id should be updated to use it (+ [keys](https://github.com/sharetribe/sharetribe/blob/master/client/app/components/sections/SearchPage/SearchPage.js#L28) and related stories/tests)